### PR TITLE
[everflow]: Make Policer test more robust

### PR DESCRIPTION
**Pull request by nazariig**
Signed-off-by: Nazarii Hnydyn <nazariig@mellanox.com>

### Description of PR

Fixed everflow Policer test: logic was made more robust

### Type of change

- [x] Bug fix
- [x] Test case(new/improvement)
- [ ] Testbed and Framework(new/improvement)

#### How did you do it?
* Changed packet generator logic: send all packets at full rate
* Added CBS bucket refill calculation
* Default tolerance is 10%

#### How did you verify/test it?
1. Deploy t1-lag
2. Run test
```bash
root@93d22705bd72:~# ./run.sh
everflow_policer_test.EverflowPolicerTest ...
Recieved 500 original packets
Received 100 mirrored packets after rate limiting
ok

----------------------------------------------------------------------
Ran 1 test in 3.423s

OK
```
3. The test will print nice user friendly assert in case of failure
```bash
root@93d22705bd72:~# ./run.sh
everflow_policer_test.EverflowPolicerTest ...
Recieved 500 original packets
Received 88 mirrored packets after rate limiting
FAIL

======================================================================
FAIL: everflow_policer_test.EverflowPolicerTest
----------------------------------------------------------------------
Traceback (most recent call last):
  File "acstests/everflow_policer_test.py", line 212, in runTest
    assert count >= self.min_range and count <= self.max_range, assert_str
AssertionError: min(90) <= count(88) <= max(110)

----------------------------------------------------------------------
Ran 1 test in 2.911s

FAILED (failures=1)
No handlers could be found for logger "dataplane"

******************************************
ATTENTION: SOME TESTS DID NOT PASS!!!

The following tests failed:
EverflowPolicerTest

******************************************
```

#### Any platform specific information?
* N/A

#### Supported testbed topology if it's a new test case?
* N/A

### Documentation 

* N/A

### CODE REVIEW

File: ansible/roles/test/files/acstests/everflow_policer_test.py [hunk 1]
```diff
@@ -152,14 +172,16 @@ def match_payload(pkt):
...
 
         count = 0
+        testutils.send_packet(self, self.src_port, self.base_pkt, count=self.NUM_OF_TOTAL_PACKETS)
         for i in range(0,self.NUM_OF_TOTAL_PACKETS):
-            testutils.send_packet(self, self.src_port, self.base_pkt)
             (rcv_device, rcv_port, rcv_pkt, pkt_time) = testutils.dp_poll(self, timeout=0.1, exp_pkt=masked_exp_pkt)
             if rcv_pkt is not None and match_payload(rcv_pkt):
                 count += 1
             elif count == 0:
                 print "The first mirrored packet is not recieved"
```
Comment (stepanblyschak) [inline ansible/roles/test/files/acstests/everflow_policer_test.py]: One small note, rather then comment.
Previous implementation had a strict check that first mirrored packet is not rate limited by the policer.
If this is OK I suggest to remove "elif count == 0" block

Comment (nazariig) [reply ansible/roles/test/files/acstests/everflow_policer_test.py]: @stepanblyschak The reason why i didn't touch it, just to have an additional informative message. But you are right, we can completely remove it.

File: ansible/roles/test/files/acstests/everflow_policer_test.py [hunk 2]
```diff
@@ -61,6 +74,13 @@ def setUp(self):
         self.src_port = int(self.test_params['src_port'])
         self.dst_mirror_ports = [int(p) for p in self.test_params['dst_mirror_ports'].split(",") if p]
         self.dst_ports = [int(p) for p in self.test_params['dst_ports'].split(",")]
+        self.meter_type = self.test_params['meter_type']
+        self.cir = int(self.test_params['cir'])
```
Comment (qiluo-msft) [inline ansible/roles/test/files/acstests/everflow_policer_test.py]: >cir [](start = 13, length = 3)

check for non-zero?

Comment (nazariig) [reply ansible/roles/test/files/acstests/everflow_policer_test.py]: @qiluo-msft done

**Pull request merged**